topics/single-cell/tutorials/scrna_batch_correction/tutorial.md

```diff
@@ -0,0 +1,529 @@
+> >
+> {: .solution}
+>
+{: .question}
+
+</div>
+
+# Clustering with Batch Correction
+
+It looks like we do need to perform batch correction on our dataset. The Scanpy and Seurat pipelines both provide tools that can reduce the technical differences between batches. If we were combining multiple datasets, we could use the same tools in the same way to perform integration (i.e. to correct for technical differences between the datasets).
+
+<div class='Scanpy' markdown='1'>
+
+SCANPY CLUSTERING with INTEGRATION
+
+</div>
+
+<div class='Seurat' markdown='1'>
+
+We will now run Seurat's batch correction tool - it's called `IntegrateLayers`, but despite the name we can use the same tool to address differences between batches as we would for integrating datasets.
+
+> <comment-title> short description </comment-title>
+>
+> {% tool Seurat Integrate %} provides several integration methods, which all perform the integration or batch correction in their own way. You might want to experiment by using one of the other methods to see how it affects the results. When you are working on your own data, it can be a good idea to try a few different integration methods to see which one produces the best results. The best integration or batch correction would be the one that eliminates the most of the technical differences between datasets or batches while producing biologically meaningful results. If we end up with completely unexpected results rather than clusters that match up well with known cell types, then we know that something has gone wrong!
+{: .comment} 
+
+> <hands-on-title> Task description </hands-on-title>
+>
+> 1. {% tool [Seurat Integrate](toolshed.g2.bx.psu.edu/repos/iuc/seurat_integrate/seurat_integrate/5.0+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Input file with the Seurat object"*: `rds_out` (output of **Seurat Run Dimensional Reduction** {% icon tool %})
+>    - *"Method used"*: `Apply integration methods with 'IntegrateLayers'`
+>        - *"Integration method to use"*: `CCA Integration`
+>        - *"Name for new dimensional reduction"*: `integrated.cca`
+>
+>    > <comment-title> short description </comment-title>
+>    >
+>    > Make sure you remember the name you've used for the new dimensional reduction - we'll be using this later instead of the PCA we produced previously.
+>    {: .comment}
+>
+{: .hands_on}
+
+It's good practice to rejoin our layers now, so that those separate layers or batches will end up back in the same layer. We don't actually need to do this now as it won't affect the clustering results, but it is important if we want to perform downstream analyses such as Differential Expression analysis.
```

Comment:
```suggestion
It's good practice to rejoin our layers now, so that those separate layers/batches will end up together. We don't actually need to do this now (as it won't affect the clustering results), but it is important if we want to perform downstream analyses such as Differential Expression analysis.
```